packages/sqlite/src/index.ts

```diff
@@ -404,8 +406,9 @@ export class SQLiteDriver extends Driver {
   }
 
   async upsert(sel: Selection.Mutable, data: any[], keys: string[]) {
-    if (!data.length) return
+    if (!data.length) return {}
     const { model, table, ref } = sel
+    const result: Driver.WriteResult = { inserted: 0, modified: 0 }
```

Comment:
```suggestion
    const result = { inserted: 0, modified: 0 }
```

Similar to the one above.